docs/installation/slack-installation.md

```
@@ -0,0 +1,15 @@
+# Install Robusta Slack Bot (3&#8203;rd party)
+
+First install [Robusta SaaS](ui-installation.md), then tag HolmesGPT in any Slack message for instant analysis.
+
+![Robusta Slack Bot powered by Holmes](../assets/RobustaSlackBot-Poweredby-Holmes.png)
+
+### Setup Slack Bot
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)